docs/source/user-guide/configs.md

```diff
@@ -65,7 +65,7 @@ Environment variables are read during `SessionConfig` initialisation so they mus
 | datafusion.execution.parquet.max_row_group_size            | 1048576                   | Sets maximum number of rows in a row group                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | datafusion.execution.parquet.created_by                    | datafusion version 32.0.0 | Sets "created by" property                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| datafusion.execution.parquet.column_index_truncate_length  | NULL                      | Sets column index truncate length                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| datafusion.execution.parquet.column_index_truncate_length  | NULL                      | Sets column index trucate length                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
I am not sure why this is required on this PR (maybe the CI didn't fully run on another PR 🤔 ) anyhow it seems a straightforward enough fix

Reply:
Hmm, isn't `truncate` a typo too?

Reply:
🤔  it is -- but that seems to have been introduced in #7244

https://github.com/apache/arrow-datafusion/pull/7244/files#diff-2f697c310af1a48521829d8bd665cf64b6415fbf88edd370efa30b1ed686354dR328

I am very confused. Let's merge this PR and I'll make another one to fix that typo

Reply:
https://github.com/apache/arrow-datafusion/pull/7874 to fix that one too